js_notification_DOM: Cypress tests added (#113)
mate-academy/js_notification_DOM: README.md

1. Replace `<your_account>` with your Github username in the link
    - [DEMO LINK](https://<your_account>.github.io/js_notification_DOM/)
2. Follow [this instructions](https://mate-academy.github.io/layout_task-guideline/)
    - Run `npm run test` command to test your code;
    - Run `npm run test:only -- -n` to run fast test ignoring linter;
    - Run `npm run test:only -- -l` to run fast test with additional info in console ignoring linter.

### Task: Create a notification

To inform the user about the success of the operation, or any errors, websites usually use messages that appear for a while on the screen.

Your task: Create function `pushNotification`, which takes coordinates of the message, title, description, and type 
(success, error and warning). `pushNotification` creates an element to display the message, appends it to the document, and removes it from the DOM after 2 seconds.
Do not write any CSS styles or HTML code in this task. You should modify only `main.js` file.

Implementation tips:

> - Print three messages: `success`, `error`, `warning` (see image below);

Reviewer: explain that function calls are already in main.js

Author: done

- The message is block element with class `notification` + class which depends on the input parameter type (`success`, `error`, `warning`); 
- The message should have a title with class `title` (prefer `h2` element);
- The message should have description (prefer tag `p`);
- Set coordinates of the message (`top` and `right`) rely on values from parameters. Use this coordinates for your
messages: 
        - `success`: 10, 10; 
        - `error`: 150, 10; 

>         - `warning`: 290, 10. 

Reviewer: is this section needed?

Author: agree. deleted

1. Replace `<your_account>` with your Github username in the link
    - [DEMO LINK](https://<your_account>.github.io/js_notification_DOM/)
2. Follow [this instructions](https://mate-academy.github.io/layout_task-guideline/)
    - Run `npm run test` command to test your code;
    - Run `npm run test:only -- -n` to run fast test ignoring linter;
    - Run `npm run test:only -- -l` to run fast test with additional info in console ignoring linter.

### Task: Create a notification

To inform the user about the success of the operation, or any errors, websites usually use messages that appear for a while on the screen.

> Your task: Create function `pushNotification`, which takes coords of future message, title, description and type 

Reviewer: ```suggestion
Your task: Create function `pushNotification`, which takes coordinates of the message, title, description, and type 
```

> (success, error and warning), appends it to the document and remove it after 2 seconds.

Reviewer: ```suggestion
(success, error and warning). `pushNotification` creates an element to display the message, appends it to the document, and removes it from the DOM after 2 seconds.
```

> You no need to write any styles or HTML code in this task. Change only `main.js` file.

Reviewer: ```suggestion
Do not write any CSS styles or HTML code in this task. You should modify only `main.js` file.
```

Implementation tips:

> - Print three messages (`success`, `error`, `warning`);

Reviewer: ```suggestion
- Print three messages: `success`, `error`, `warning` (see image below);
```

- The message is block element with class `notification` + class which depends on the input parameter type (`success`, `error`, `warning`); 
- The message should have a title with class `title` (prefer `h2` element);
- The message should have description (prefer tag `p`);
- Set coordinates of the message (`top` and `right`) rely on values from parameters. Use this coordinates for your
messages: 
        - `success`: 10, 10; 
        - `error`: 150, 10; 

>         - `warning`: 290, 10. 

Reviewer: who calls the pushNotification method?  Should we add the call in solution template?

Author: calls were added to the solution template

Hint: use [setTimeout](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout) for execute a function or specified piece of code once the timer expires.

It's an expected result of your job:

> ![Result](./src/images/result_new.png)

Reviewer: why rename the file? Just replace the content.

You no need to write any styles or HTML code in this task. Change only `main.js` file.

Implementation tips:
- Print three messages (`success`, `error`, `warning`);
- The message is block element with class `notification` + class which depends on the input parameter type (`success`, `error`, `warning`); 
- The message should have a title with class `title` (prefer `h2` element);
- The message should have description (prefer tag `p`);

> - Set coordinates of the message (top and right) rely on values from parameters. You can find coordinates in tests.

Reviewer 2: Не треба, щоб вони шукали в тестах все має бути зрозумілим з умови, чи бути у вхідних параметрах, якщо мова йде про функцію

Author: відредагував умову, щоб було зрозуміліше
